sdks/python/apache_beam/yaml/examples/simple_filter_and_combine.yaml

```diff
@@ -0,0 +1,56 @@
+# coding=utf-8
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the#  Row(word='License'); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an#  Row(word='AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# The following example reads mock transaction data from resources/products.csv,
+# performs a simple filter for "Electronics", then calculates the revenue and
+# number of products sold for each product type.
+pipeline:
+  transforms:
+    - type: ReadFromCsv
+      name: ReadInputFile
+      config:
+        path: resources/products.csv
+    - type: Filter
+      name: FilterWithCategory
+      input: ReadInputFile
+      config:
+        language: python
+        keep: category == "Electronics"
+    - type: Combine
+      name: CountNumberSold
+      input: FilterWithCategory
+      config:
+        group_by: product_name
+        combine:
+          num_sold:
+            value: product_name
+            fn: count
+          total_revenue:
+            value: price
+            fn: sum
+    - type: WriteToCsv
+      name: WriteOutputFile
+      input: CountNumberSold
+      config:
+        path: output
+
+options:
+  yaml_experimental_features: Combine
+
+# Expected:
```

Comment:
[README.md](https://github.com/apache/beam/blob/9add040efc69c516b5462668ac923979cc38a6d0/sdks/python/apache_beam/yaml/examples/README.md) can we update this?

Reply:
Update it how?

Reply:
Add this to the list?

Reply:
It's not really a list, it is just saying there are examples that highlight specific transforms in the `transforms/` subdirectory and to start with Wordcount in the root directory. All other examples are also in the root directory, but I don't think they need calling out. Their purpose is described in the header of each